raisinbread/RB_files/RB_Config.sql

```diff
@@ -34,6 +34,7 @@ INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (34,38,'/data/uploads/')
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (35,40,'main.css');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (36,41,'25');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (37,44,'localhost');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (38,45,'http://localhost');
```

Comment:
Should the `http://` be necessary? I'm not sure what the uses is for this row.

Reply:
yes
thats what was already there i just restored it because all tests failed when URL was removed

Reply:
@ridz1208 I mean the http part. What if the developer setups https instead. I would think just localhost would work? Unless the value is being used for something specific but I don't know.

Reply:
afaik its being used for tests and any dev should change this value eitherway and if you try to use http but your SSL is setup it wont even load

you can ignore that for now